src/tests/ftest/vmd/led.py

```diff
@@ -0,0 +1,161 @@
+"""
+  (C) Copyright 2020-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import json
+import time
+from osa_utils import OSAUtils
+from avocado.utils import process
+
+
+class VmdLedStatus(OSAUtils):
+    """
+    Test Class Description: This class methods to get
+    the VMD LED status.
+    """
+    # pylint: disable=too-many-instance-attributes,too-many-ancestors
+    def setUp(self):
+        super().setUp()
+        self.targets = self.params.get("targets", "/run/server_config/servers/0/*")
+        self.dmg = self.get_dmg_command()
+        self.dmg.hostlist = self.hostlist_servers[0]
+
+    def get_nvme_device_ids(self):
+        """Get the list of nvme device-ids.
+        Returns: List of uuid
+        """
+        self.dmg.json.value = True
+        try:
+            result = self.dmg.storage_query_list_devices()
+        except process.CmdError as details:
+            self.fail("dmg command failed: {}".format(details))
+        finally:
+            self.dmg.json.value = False
+
+        data = json.loads(result.stdout_text)
+        resp = data['response']
+        if data['error'] or len(resp['host_errors']) > 0:
+            if data['error']:
+                self.fail("dmg command failed: {}".format(data['error']))
+            else:
+                self.fail("dmg command failed: {}".format(resp['host_errors']))
+        uuid = []
+        for value in list(resp['host_storage_map'].values()):
+            if value['storage']['smd_info']['devices']:
+                for target in range(self.targets):
+                    uuid.append(value['storage']['smd_info']['devices'][target]['uuid'])
+        return uuid
+
+    def get_led_status_value(self, device_id=None):
+        """Get LED Status value.
+
+        Args:
+            device_id (str): Device UUID
+        Returns:
+            dmg LED status information
```

Comment:
There should be a type assignment here.  This returns a dictionary, correct?
```suggestion
            dict: dmg LED status information
```

Reply:
Yes. Will update it.